docs/faq.md

```diff
@@ -49,12 +69,8 @@ spec:
   - name: local.csi.openebs.io
     nodeID: k8s-node-1
     topologyKeys:
-    - beta.kubernetes.io/arch
-    - beta.kubernetes.io/os
-    - kubernetes.io/arch
     - kubernetes.io/hostname
```

Comment:
hostname should not be there as we have just mentioned `openebs.io/rack` only in ALLOWED_TOPOLOGIES. or   Probably better to add `kubernetes.io/hostname` here https://github.com/openebs/lvm-localpv/pull/110/files#diff-57cdcc6b6701a7ce3b3a8f8c0366e0e611e6fb1a1b8dd7cd29e3263b3e064c8bR32.

Reply:
I add "kubernetes.io/hostname" and "openebs.io/nodename" as default topology keys(see pkg/driver/agent.go) which are default labels generated by k8s and lvm provisioner.